boards/arm/nucleo_l552ze_q/CMakeLists.txt

```diff
@@ -16,49 +16,50 @@ if (CONFIG_BUILD_WITH_TFM)
 		set(TFM_IMAGE_VERSION_NS 0.0.0+0)
 	endif()
 
-	set(PREPROCESSED_FILE "${CMAKE_BINARY_DIR}/tfm/image_macros_preprocessed")
-	set(TFM_MCUBOOT_DIR "${ZEPHYR_BASE}/../modules/tee/tfm/trusted-firmware-m/bl2/ext/mcuboot")
+	set(PREPROCESSED_FILE_S "${CMAKE_BINARY_DIR}/tfm/bl2/ext/mcuboot/CMakeFiles/signing_layout_s.dir/signing_layout_s.o")
+	set(PREPROCESSED_FILE_NS "${CMAKE_BINARY_DIR}/tfm/bl2/ext/mcuboot/CMakeFiles/signing_layout_ns.dir/signing_layout_ns.o")
+	set(TFM_MCUBOOT_DIR "${ZEPHYR_TFM_MODULE_DIR}/trusted-firmware-m/bl2/ext/mcuboot")
 
 	# Configure which format (full or hash) to include the public key in
 	# the image manifest
-	set(TFM_PUBLIC_KEY_FORMAT "hash")
+	set(TFM_PUBLIC_KEY_FORMAT "full")
 
 	#Create and sign for concatenated binary image, should align with the TF-M BL2
 	set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
 
 		#Sign secure binary image with public key
-		COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/imgtool.py
-		ARGS sign
-			 --layout ${PREPROCESSED_FILE}_s.c
+		COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/wrapper/wrapper.py
+			 --layout ${PREPROCESSED_FILE_S}
 			 -k ${CONFIG_TFM_KEY_FILE_S}
 			 --public-key-format ${TFM_PUBLIC_KEY_FORMAT}
 			 --align 1
 			 -v ${TFM_IMAGE_VERSION_S}
+			 --pad
+			 --pad-header
 			 ${ADD_NS_IMAGE_MIN_VER}
-			 ${ADD_SECURITY_COUNTER_S}
+			 -s auto
 			 -H 0x400
-			 ${CMAKE_BINARY_DIR}/tfm/install/outputs/STM_NUCLEO_L552ZE_Q/tfm_s.bin
+			 ${CMAKE_BINARY_DIR}/tfm/install/outputs/STM/NUCLEO_L552ZE_Q/tfm_s.bin
```

Comment:
Should is be lower case here to match TFM_TARGET_PLATFORM value ?

Reply:
Nope, this should be compliant with upstream TFM - this is how it looks like in the new build system.

Reply:
ok, txs